Gemfile

```diff
@@ -6,6 +6,6 @@ if !ENV['TRAVIS']
   gem 'yard',   require: false
 end
 
-gem 'lotus-utils', '~> 0.5', require: false, github: 'lotus/utils', branch: '0.5.x'
+gem 'lotus-utils', '~> 0.5', require: false, github: 'theocodes/utils', branch: 'same-strings-join-fix'
```

Comment:
@theocodes your lotus/utils PR was merged. Can you change this line with lotus/utils master branch? thanks ;)